index.html

```diff
@@ -297,11 +298,95 @@ <h2>
           For a given <a>document</a>, the <a>current orientation type</a> and
           the <a>current orientation angle</a> are strongly linked in the sense
           that for any given type, there will be a specific angle associated.
-          However, the <a>user agent</a> can associate <code>*-primary</code>
-          and <code>*-secondary</code> values at will. For example, if
-          <code>90</code> is associated with <code>landscape-primary</code> and
-          <code>270</code> with <code>landscape-secondary</code> for one
-          <a>document</a> , another one MAY get the opposite relationship.
+        </p>
+        <p>
+          One primary orientation will always be determined by the natural
+          orientation of the device and this will then determine the secondary
+          value of its related orientation.
+        </p>
+        <p>
+          For example a device held in its natural portrait orientation would
+          have a current orientation of <a>portrait-primary</a> and its
+          <a>portrait-secondary</a> orientation would be its position when
+          rotated 180°.
+        </p>
+        <p>
+          The <a>user agent</a> can associate the other <code>*-primary</code>
+          and <code>*-secondary</code> values at will. For example, it can be
+          based on the device preferred angles, the user's preferred
+          orientations or the current orientation when the application starts.
+        </p>
+        <table>
+          <thead>
+            <tr>
+              <th>
+                Natural Orientation
+              </th>
+              <th>
+                Primary Orientation 1
+              </th>
+              <th>
+                Secondary Orientation 1
+              </th>
+              <th>
+                Primary Orientation 2
+              </th>
+              <th>
+                Secondary Orientation 2
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                Portrait
+              </td>
+              <td>
+                Portrait-primary<br>
```

Comment:
nit: link "portrait-primary" etc... 

Using a `<br>` seems a little odd... we probably don't need to fix this now, but we should think about maybe a better way of arranging the cells. It doesn't matter too much tho.